hack/jenkins/e2e.sh

```diff
@@ -180,12 +176,6 @@ GCE_PARALLEL_FLAKY_TESTS=(
     "Namespaces.*should\sdelete\sfast"
     "PD"
     "ServiceAccounts"
-    "Services.*change\sthe\stype"
-    "Services.*functioning\sexternal\sload\sbalancer"
-    "Services.*identically\snamed"
-    "Services.*release.*load\sbalancer"
-    "Services.*endpoint"
```

Comment:
7324 was a "All nodes should be ready" error and a 10 minute timeout (must be something GCE-related).

7477 were a bunch of related timeouts.

I still assert this test does not belong in flaky